source/kernel/kernel.go

```diff
@@ -170,7 +170,9 @@ func parseKconfig() (map[string]bool, error) {
 	for _, line := range lines {
 		if m := re.FindStringSubmatch(string(line)); m != nil {
 			if m[2] == "y" || m[2] == "m" {
-				kconfig[m[1]] = true
+				kconfig[m[1]] = "true"
+			} else {
+				kconfig[m[1]] = strings.Trim(m[2], `"`)
```

Comment:
Generally ok, however here it might be good to add a bit more validation for value, beside just stripping `"`

Reply:
Validation of label values implemented in #199. I think that should be enough, WYDT?

Reply:
in theory should be enough. my comment was more about general practice of validating input close to the point of source of information.

Reply:
I wouldn't want to do duplicate regexp matching in multiple places. Checking the length makes sense so I added that.